app/views/groups/show.html.slim

```diff
@@ -28,12 +28,12 @@
     .row-label = t('groups.show.users')
 
     .row-value
-      - @group.members.each do |member|
+      - (@group.admins + @group.confirmed_members).each do |member|
```

Comment:
While I initialily thought it would be nice to show the together, I think it would be cleaner (and less redundant) to split `confirmed_members` and `admins`

Reply:
Okay, then let's do that :+1: